docs/articles/documentation/test-api/selecting-page-elements/selectors.md

```diff
@@ -84,28 +90,26 @@ You can initialize a selector with any of these objects.
     import { Selector } from 'testcafe';
 
     // This selector is created from a function that returs all elements of a specified class.
-    // The selector returns the third such element since its `index` option is set to 2.
-    const thirdElemWithClass = Selector(cl => document.getElementsByClassName(cl), {
-        index: 2
+    // The selector additionally requires the returned element to become visible.
+    const elemWithClass = Selector(cl => document.getElementsByClassName(cl), {
+        visibilityCheck: true
     });
 
-    // This selector is created based on the previous selector and inherits its initializer,
-    // but overwrites the `index` parameter to return the fourth element.
-    const fourthElemWithClass = Selector(thirdElemWithClass, { index: 3 });
+    // This selector is created based on the previous selector and inherits
+    // its initializer, but overwrites the `visibilityCheck` parameter.
+    const fourthElemWithClass = Selector(thirdElemWithClass, { visibilityCheck: false });
```

Comment:
[Kazakov: the previous selector was renamed, so rename the current one.]
const hiddenElementsWithClass = Selector(elementsWithClass, { visibilityCheck: false });